K8s-deployment/Charts/keycloak/values.yaml

```diff
@@ -578,37 +487,49 @@ initContainers: {}
 ##       - name: portname
 ##         containerPort: 1234
 ##
-sidecars: {}
+sidecars: []
 
 ## @section Exposure parameters
+##
 
 ## Service configuration
 ##
 service:
   ## @param service.type Kubernetes service type
   ##
-  type: ClusterIP
-  ## @param service.port Service HTTP port
+  type: LoadBalancer
```

Comment:
Why is the service LoadBalancer? It should be clusterIP.
ingress->clusterIp service